docs/cookbook/orbit-controls.md

```diff
@@ -112,7 +114,12 @@ The `cientos` package provides a component called `<OrbitControls />` that is a
 The nicest part? You don't need to extend the catalog or pass any arguments.  
 It just works. 💯
 
-```vue
+```vue {3,12}
+<script setup lang="ts">
+import { TresCanvas } from '@tresjs/core'
+import { OrbitControls } from '@trejs/cientos'
```

Comment:
> import { OrbitControls } from '@trejs/cientos'

There's a missing 's' in `@trejs`.

> import { OrbitControls } from '@tresjs/cientos'

Also in the Spanish/German docs.

Reply:
LOL, new library